root/account/applications/Register.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import Layout from '../../layout';
+import * as manifest from '../../static/manifest';
+import {l} from '../../static/scripts/common/i18n';
+
+import RegisterApplicationForm from './RegisterForm';
+import type {RegisterApplicationFormPropsT} from './RegisterForm';
+
+const RegisterApplication = (props: RegisterApplicationFormPropsT) => (
+  <Layout fullWidth title={l('Applications')}>
+    <h1>{l('Register Application')}</h1>
+    <RegisterApplicationForm {...props} />
+    <script async="async" src={manifest.pathTo('/scripts/account/applications/register.js')} />
```

Comment:
Hm, that's always a 404 for me. You'll need to add something like this to the gulpfile so that it's bundled properly:

```JavaScript
runYarb('account/applications/register.js', function (b) {
  b.external(commonBundle);
});
```

There's also a `manifest.js` utility you can use to replace the `<script/>`, to get the correct path:

```JavaScript
{manifest.js('account/applications/register', {async: 'async'})}
```

Reply:
Oops, I got it right first, then removed `runYarb` and it continued to work, probably due to some cache. That explained why the handler for `oauth_redirect_uri` didn’t work then at that time.